### ISSUE TITLE
Floating point execption (core dumped)

### ISSUE THREAD

**Issue by zymy-chen** [hpcaitech/Open-Sora#656]
Same docker（hpcaitech/pytorch-cuda:2.1.0-12.1.0）, I'm running opensora-v1.2 with H20 and it comes up Floating point execption, but A100 works fine.

configs/opensora-v1-2/inference/sample.py:
resolution = "240p"
aspect_ratio = "9:16"
num_frames = 51
fps = 24
frame_interval = 1
save_fps = 24

save_dir = "./samples/samples/"
seed = 42
batch_size = 1
multi_resolution = "STDiT2"
dtype = "bf16"
condition_frame_length = 5
align = 5

model = dict(
    type="STDiT3-XL/2",
    from_pretrained="xxx",
    qk_norm=True,
    enable_flash_attn=True,
    enable_layernorm_kernel=False,
)
vae = dict(
    type="OpenSoraVAE_V1_2",
    from_pretrained="xxx",
    micro_frame_size=17,
    micro_batch_size=4,
)
text_encoder = dict(
    type="t5",
    from_pretrained="xxx",
    model_max_length=300,
)
scheduler = dict(
    type="rflow",
    use_timestep_transform=True,
    num_sampling_steps=30,
    cfg_scale=7.0,
)

aes = 6.5
flow = None


python scripts/inference.py configs/opensora-v1-2/inference/sample.py --prompt-path assets/texts/t2v_sora.txt
[2024-08-05 02:22:09] Building models...
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:15<00:00,  7.69s/it]
[2024-08-05 02:22:44] loading pretrained model: /home/gwn/OpenSora-STDiT-v3/model.pth
[2024-08-05 02:22:44] Missing keys: []
[2024-08-05 02:22:44] Unexpected keys: []
  0%|                                                                                                                                                                                                                                         | 0/48 [00:00<?, ?it/s]Floating point exception (core dumped)

**Comment by pokaaa** [comment]
Have u solved this problem? I have met the same problem on H20...

**Comment by zymy-chen** [comment]
> Have u solved this problem? I have met the same problem on H20...

I tried to update cuda but it didn't work

**Comment by qinziqiao** [comment]
same problem

**Comment by Yanjia0** [comment]
Hello， have you tried to upgrade driver together with cuda version? This may help to solve the problem.
